src/manifest.json

```diff
@@ -54,25 +55,25 @@
     },
     "default_popup": "popup.html"
   },
-  "options_page": "options.html",
+  "options_ui": {
+    "page": "options.html",
+    "open_in_tab": true
+  },
   "icons": {
     "16": "img/ic_suspendy_16x16.png",
     "32": "img/ic_suspendy_32x32.png",
     "48": "img/ic_suspendy_48x48.png",
     "128": "img/ic_suspendy_128x128.png"
   },
-  "web_accessible_resources": [
-    "suspended.html"
-  ],
   "content_security_policy": "script-src 'self'; object-src 'self'; child-src 'self'; connect-src 'self'; img-src 'self' data: chrome:; style-src 'self'; default-src 'self'",
-  "incognito": "split",
+  "incognito": "not_allowed",
```

Comment:
I could not make it work in private mode in FF, the easiest workaround would be to split manifests and package different ones for Chrome and FF